src/library_idbfs.js

```diff
@@ -83,7 +83,7 @@ mergeInto(LibraryManager.library, {
         callback(null, db);
       };
       req.onerror = (e) => {
-        callback(this.error);
+        callback(req.error || 'unknown error');
```

Comment:
Ooops, this was my fault.  Can't we instead switch back to regular `function` here?

Reply:
Yes it's possible, as well as using `e.target.error` instead of referring to a closure variable if that's what you don't like.

Reply:
What do you think is more correct/recommenced way?

Reply:
`event.target.error` is already used this way in ["IDBStore.js"](https://github.com/emscripten-core/emscripten/blob/cd3596b5d922e2f84956bdf005bb150277f5682d/src/IDBStore.js#L48) so that would be coherent to do the same here.

Reply:
OK sounds good.

Reply:
Done.